.github/workflows/docgen.build.deploy.dev.yaml

```diff
@@ -0,0 +1,43 @@
+name: Build docgen
+
+on: 
+  pull_request:    
+  push:
+    branches:
+      - develop
+    path:
+      - services/core-api/**
+      - .github/workflows/docgen.build.yaml
+
+env:
+  INITIAL_TAG: latest
+  TAG: dev
+  NAME: docgen
+  CONTEXT: services/core-api/
+  image: docker.io/bcgovimages/common-document-generation-service
```

Comment:
We do we need to build on top of this?

Reply:
This file is essentially just taking the Dockerfile and pulling it to build. A little redundant if we don't update it, but I figured this way gave us the opportunity to have a specified way to build this service, as well as specify which version we pull.

Reply:
Fair enough. I'll just remove the references for this stuff and put the image into the kustomize deployments then.